### ISSUE TITLE
Upload multiple image issue

### ISSUE THREAD

**Issue by HardikKardani** [apollographql/apollo-ios#1264]
Hello

I am trying to upload user profile picture and cover photo using below code..

issue:- every time i am getting json formate error and status code 400. but if i only upload single photo and other image as nill then it works fine.

```
if let imageData = imgProfilePicture.image!.jpegData(compressionQuality: 0.7){
            let fileFieldName   = "pic"
            let file            = GraphQLFile(fieldName: fileFieldName, originalName: "fileFieldName.jpg", mimeType: "image/jpg", data: imageData)
            
            let imageCoverData = imgCoverPhoto.image!.jpegData(compressionQuality: 0.7)
            let coverPhotoFieldName   = "coverPhoto"
            let coverPhotoFile            = GraphQLFile(fieldName: coverPhotoFieldName, originalName: "coverPhoto.jpg", mimeType: "image/jpg", data: imageCoverData!)
            
            Network.shared.apollo.upload(operation: UpdateUserMutation(email:"test@gmail.com", pic: "0", coverPhoto: "1"), files: [file, coverPhotoFile])
}
```

is there any issue in passing data or what?

**Comment by designatednerd** [comment]
I think this may be related to #1081 - I'm going to try to land that by the end of this week but for now I think the best workaround is to upload one photo at a time.

**Comment by HardikKardani** [comment]
@designatednerd  thank you for quick update.

**Comment by HardikKardani** [comment]
@designatednerd is there any update ?

**Comment by designatednerd** [comment]
Was not able to get this done last week and then have been having [some technical difficulties](https://twitter.com/designatednerd/status/1275176538963574791) this week, but it's still on my list to land that PR!

**Comment by designatednerd** [comment]
Please see #1279 for what should be a solution to this.

**Comment by designatednerd** [comment]
This fix has shipped with `0.29.0`. If you are still having problems after upgrading to that version, please open a new issue with details, noting that you are using that version. Thank you!

**Comment by HardikKardani** [comment]
Thanks @designatednerd 

Will check and inform you.
